styles/colors-carbon.less

```diff
@@ -169,7 +170,7 @@
 // RadioItem
 // ---------------------------------------
 @agate-radioitem-focus-color: @agate-item-color;
-@agate-radioitem-separator-border-color: @agate-divider-border-color;
+@agate-radioitem-separator-border-color: fadeout(@agate-text-color, 30%);
```

Comment:
I think this should be `@agate-heading-border-color`
(and in all the other `.less` files, this change should be made)

Reply:
@sugardave is correct. Setting this to `@agate-heading-border-color` preserves the relationships between the variables rather than assigning two separate values which happen to be the same value.